cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -148,9 +156,26 @@ public void execute() throws CommandException {
             }
         }
 
+        writePullRunFileIfNeeded();
+
         consoleWriter.fg(Color.GREEN).newLine().a("Finished").println(2);
     }
 
+    void initPullRunName() {
+        if (recordPullRun) {
+            pullRunName = UUID.randomUUID().toString();
+        }
+    }
+
+    void writePullRunFileIfNeeded() {
+        if (recordPullRun) {
+            Path pullRunOuputFile = commandDirectories.getTargetDirectoryPath().resolve("pull-run-name.txt");
```

Comment:
Should the full output file path be explicitly be passed in as a param?

Reply:
see previous comment.

Code wise I need to share the code at least